recipes/miktex-portable/post-link.bat

```diff
@@ -0,0 +1,5 @@
+rem packages for nbconvert
```

Comment:
More or less because currently (and I don't see that changed until a solution is added to nbconvert in continuums packages) nbconvert will not depend on a latex environment. as fasr as I can identify the packages, installing them seems to be fine for more than nbconvert (e.g. knitr/RMarkdown in the r packages) and I would add missing latex packages for these packages here, too.